### ISSUE TITLE
[BUG] seednode binary dependency on shared libraries: libwasmer_linux_amd64.so

### ISSUE THREAD

**Issue by frankf1957** [ElrondNetwork/elrond-go#2927]
<details>
<summary>Executing details</summary>
<pre>
Application version string: SeedNode CLI App version v0.0.1
Hosting VPS: Hetzner AX51-NVMe 
</pre>
</details>

I am installing seednode on a new Ubuntu 18.04.5 system,that does not have the Elrond node installed.  When I try to start the seednode binary, it fails with the following error:

$ ./seednode 
./seednode: error while loading shared libraries: libwasmer_linux_amd64.so: cannot open shared object file: No such file or directory

$ ./seednode --version
./seednode: error while loading shared libraries: libwasmer_linux_amd64.so: cannot open shared object file: No such file or directory

seednode binary should not have a dependency on the libwasmer_linux_amd64 shared object library.

**Comment by bogdan-rosianu** [comment]
Hello! Thank you for opening this issue.
Did you run `go build` before?

**Comment by frankf1957** [comment]
Hi, here is what I did.

The seednode binary is an artifact of the elrond node install.  After a successful node install using the elrond-go-scripts-{testnet,devnet,mainnet}, it is found at: elrond-utils/seednode.

I copied the seednode binary at: elrond-utils/seednode, as well as the seednode configuration file at: go/src/github.com/ElrondNetwork/elrond-go/cmd/seednode/config/p2p.toml, to a newly installed server that did not have any Elrond nodes or tools installed.  

On the new server, I modified the p2p.toml file to set the Seed =, ProtocolID =, and InitialPeerList = to appropriate values for mainnet.  

When I started my seednode, I encountered the error - "error while loading shared libraries: libwasmer_linux_amd64.so: cannot open shared object file: No such file or directory".

If I run ldd on the seednode binary, I can see that there is a reference to  libwasmer_linux_amd64.so, and that reference is the fully qualified PATH of the user that built the Elrond node software.  The result of this is that the seednode binary is not portable; it cannot be installed an run on systems that do not have the Elrond node installed.

`~/elrond-utils$ ldd seednode
	linux-vdso.so.1 (0x00007ffc4d13f000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f3aa4e5c000)
	libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f3aa4ad3000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f3aa4735000)
	libwasmer_linux_amd64.so => /home/elrond/go/pkg/mod/github.com/!elrond!network/arwen-wasm-vm@v1.1.1/wasmer/libwasmer_linux_amd64.so (0x00007f3aa42a2000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f3aa408a000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f3aa3c99000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f3aa507b000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f3aa3a95000)
	librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f3aa388d000)`
